cumulus/parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/xcm_config.rs

```diff
@@ -330,6 +330,7 @@ impl pallet_xcm::Config for Runtime {
 	type XcmExecutor = XcmExecutor<XcmConfig>;
 	type XcmTeleportFilter = Everything;
 	type XcmReserveTransferFilter = Nothing; // This parachain is not meant as a reserve location.
+										 // TODO: flavor?
```

Comment:
```suggestion
```

I don't think this will differ - BH runtimes flavors have same `RuntimeCall` ultimately..

Reply:
sorry, forgot to remove it, removed